### PR TITLE
Migrate cluster instructions to JOIN command

### PR DESCRIPTION
**Pull request by JoeReid**
The release notes need to be written (IDK all the changes intimately), but here is a start at the docs?
Anything I'm missing?

### CODE REVIEW

File: _docs/install/prerequisites/clusterdiscovery.md [hunk 1]
```diff
@@ -45,20 +51,27 @@ $ storageos cluster create --size 3
...
+
+### Option 3: Doing both
 
-Alternatively, provide an explicit list of hostnames and IP addresses via the
-`INITIAL_CLUSTER` environment variable.
+These methods are not mutually exclusive, and in some situations it may be desirable to use both.
 
 ```bash
-INITIAL_CLUSTER=storageos-1=http://172.28.128.3:5707,storageos-2=http://172.28.128.9:5707,storageos-3=http://172.28.128.15:5707
+JOIN=d53e9fae-7436-4185-82ea-c0446a52e2cd,172.28.128.3,172.28.128.9
```
Comment (oicheryl) [inline _docs/install/prerequisites/clusterdiscovery.md]: Clarify what order these get tried in.

Comment (rusenask) [reply _docs/install/prerequisites/clusterdiscovery.md]: from left to right

Comment (JoeReid) [reply _docs/install/prerequisites/clusterdiscovery.md]: I have added clarification

**Comment by rusenask** [comment]
It would be nice to see an example of joining a single node (not just existing 2 nodes)

File: _docs/install/prerequisites/clusterdiscovery.md [hunk 2]
```diff
@@ -45,20 +51,28 @@ $ storageos cluster create --size 3
...
 are registered, StorageOS will start up.
 
 
-### Option 2: Specify hostnames and IP addresses
+### Option 2: Specifying IP addresses
+
+An alternative to the discovery service is providing an explicit list of IP addresses.
+
+```bash
+JOIN=storageos-1=172.28.128.3,172.28.128.9,172.28.128.15
```
Comment (oicheryl) [inline _docs/install/prerequisites/clusterdiscovery.md]: @JoeReid Remove "storageos-1=" here.

**Pull request merged**